Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -28,8 +28,12 @@ internal final class PaymentMethodsViewController: UIViewController {
       title: Strings.discovery_favorite_categories_buttons_edit(),
       style: .plain,
       target: self,
-      action: nil
+      action: #selector(edit)
     )
+
+    self.dataSource.deletionHandler = { [weak self] creditCard in
+      self?.viewModel.inputs.didDelete(creditCard)
```

Comment:
This flow is a little awkward but we are only told about the deletion via the `UITableViewDataSourceDelegate` 🤔

Reply:
Yeah,  another option would be to create a delegate, but I think it would be weird.